pkg/apiserver/rest/usecase/authentication.go

```diff
@@ -288,6 +267,84 @@ func (a *authenticationUsecaseImpl) GetDexConfig(ctx context.Context) (*apisv1.D
 	}, nil
 }
 
+func (a *authenticationUsecaseImpl) UpdateDexConfig(ctx context.Context) error {
```

Comment:
Who will call this function?

Reply:
When adding or deleting a dex connector, this function will be called.